chat-plugins/uno.js

```diff
@@ -452,6 +452,9 @@ exports.commands = {
 			if (!this.can("minigame", null, room)) return;
 			if (room.unoDisabled) return this.errorReply("UNO is disabled for this room.");
 			if (room.game) return this.errorReply("There is already a game in progress in this room.");
+			let num = (target === '') ? 3 : parseInt(target);
+			if (!Number.isInteger(num)) return this.errorReply("Please enter a valid player cap.");
```

Comment:
because ``parseInt`` rounds, this really isn't needed... you could check if it's not a number, I guess...